### PR TITLE
added onlyLetters option, and updated the readme. (#22)

### PR DESCRIPTION
**Pull request by aawgit**

### CODE REVIEW

**Comment by strdr4605** [comment]
Thank you for contribution! Do you mind creating some tests for this option?!

File: README.md [hunk 1]
```diff
@@ -68,8 +74,9 @@ This function receives a string input and converts it to mOcKiNgCaSe.
 | Param | Type | Default | Description |
 | --- | --- | --- | --- |
 | input | <code>string</code> \| <code>Array.&lt;string&gt;</code> |  | string or array of strings to be converted |
-| [options] | <code>object</code> | <code>{random: false}</code> | options for converting |
+| [options] | <code>object</code> | <code>{random: false} {onlyLetters: false}</code> | options for converting |
```
Comment (strdr4605) [inline README.md]: There is a a object with keys **random** and **onlyLetters**
```suggestion
| [options] | <code>object</code> | <code>{random: false, onlyLetters: false}</code> | options for converting |
```

File: README.md [hunk 2]
```diff
@@ -81,5 +88,6 @@ This function receives a string input and outputs a message to the console in mO
 | Param | Type | Default | Description |
 | --- | --- | --- | --- |
 | input | <code>string</code> \| <code>Array.&lt;string&gt;</code> |  | string or array of string to be converted |
-| [options] | <code>object</code> | <code>{random: false}</code> | options for converting |
+| [options] | <code>object</code> | <code>{random: false}  {onlyLetters: false}</code> | options for converting |
```
Comment (strdr4605) [inline README.md]:
```suggestion
| [options] | <code>object</code> | <code>{random: false, onlyLetters: false}</code> | options for converting |
```

File: index.js [hunk 3]
```diff
@@ -1,26 +1,37 @@
-'use strict';
+"use strict";
 
 /**
  * This function receives a string input and converts it to mOcKiNgCaSe.
  * @param {(string | string[])} input - string or array of strings to be converted
  * @param {object} [options={random: false}] - options for converting
```
Comment (strdr4605) [inline index.js]: can you please add `onlyLetters` also here
```suggestion
 * @param {object} [options={random: false, onlyLetters: false}] - options for converting
```

**Comment by strdr4605** [comment]
If you want, you can update `index.d.ts` with `onlyLetters` for typescript

**Comment by strdr4605** [comment]
Another pull request was merged, can you rebase and fix the conflicts

**Comment by aawgit** [comment]
> Another pull request was merged, can you rebase and fix the conflicts

This is my first contribution to a project. Would you please let me know how to do this? Thanks.

**Comment by strdr4605** [comment]
@aawgit 
1. you need to update the forked repo with changes from main repo. Take a look at https://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository
2. After you `rebase`d your `master` branch. Go to `onlyLetters` branch and `rebase` to your local `master`
3. You will have conflicts because on `onlyLetters` you modified some lines which were modified also on `upstream/master`. Resolve conflicts by accepting all `master` changes then apply your changes.
4. Apply also changes what I requested in review. 
5. Take a look at `test.js` test examples and try to create some tests for `onlyLetters` option. Run test using `npm test`

Let me know if you need more help!

**Comment by strdr4605** [comment]
Actually you can `rebase` `onlyLetters` on `upstream/master`. It may look like:
```bash
git remote add upstream https://github.com/strdr4605/mockingcase.git
git fetch upstream
git checkout onlyLetter
git rebase upstream/master
```
next fix conflicts, commit and
```bash
push origin -f onlyLetter
```

**Comment by aawgit** [comment]
> Actually you can `rebase` `onlyLetters` on `upstream/master`. It may look like:
> 
> ```shell
> git remote add upstream https://github.com/strdr4605/mockingcase.git
> git fetch upstream
> git checkout onlyLetter
> git rebase upstream/master
> ```
> 
> next fix conflicts, commit and
> 
> ```shell
> push origin -f onlyLetter
> ```

Many thanks.

**Comment by aawgit** [comment]
> LGTM, Will merge soon. @aawgit can you fix last comment in Readme line 122!?

Yes, I will. Sorry it was overlooked. How do I push it?

**Comment by strdr4605** [comment]
> Yes, I will. Sorry it was overlooked. How do I push it?

Just make another commit with changes and push it. After you will have 3 commits. When merging I will **Squash** them into 1. 
If you want, you can try to **Square** them yourself. Take a look at https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history

**Pull request merged**